src/photon/photon_scheduler.h

```diff
@@ -50,4 +50,34 @@ void process_plasma_notification(event_loop *loop,
                                  void *context,
                                  int events);
 
+/**
+ * Reconstruct an object. This triggers a single reexecution of the task that
+ * originally created the object, even if the object does exist on some node.
```

Comment:
If the object table says that the object is present on some nodes, then actually this does not trigger any reexecution, right?